contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,132 @@
+---
+date: 2024-02-05
+title: "How we built our onboarding email flow (with actual performance data)"
+rootPage: /blog
+category: Inside PostHog
+author: ["joe-martin"]
+featuredImage: ../images/blog/blog_onboarding.jpg
+featuredImageType: full
+---
+
+Email is a powerful tool for getting users to engage with your product, but it can be difficult to know how the best companies structure their onboarding email flows. You end up having to sign up to a bunch of them, then waiting and letting your inbox get flooded. 
+
+Often, you end up getting awful sales rep outreach emails from them _months_ later too. That's important because it's probably the root cause for the conventional wisdom that developers do not respond well to email marketing. Turns out, they just don't respond well to _bad_ email marketing. 
+
+We don’t do those sort of outreach emails. This is what we do instead, and how we got there over 5 major iterations. We think it works and we've included performance data so you can judge for yourself.
+
+## Onboarding 1.0: The one where it started costing lots
+
+Back in 2022 our onboarding flow was powered by Mailchimp and was very, _very_ simple. In fact, nobody actually knew what it looked like. So, I mapped it in [an RFC](https://github.com/PostHog/posthog.com/issues/3202).
+
+![Onboarding 1.0](../images/blog/onboarding/onboarding_1.png)
+<caption>The first map we did of what our email onboarding flow looked like</caption>
+
+We didn’t do welcome emails for new users, and even our monthly changelog emails were only sent to 660 opted-in users. I decided to change this and imported 30,000+ users into that email list. The move was covered by our T&Cs and, because it was one email a month about actual product updates, we figured nobody would mind. The good news was that we were right and didn't get a single complaint and only a handful of unsubscribes.
+
+The bad news is that it sent our Mailchimp costs skyrocketing. 
```

Comment:
```suggestion
The bad news is that it sent our Mailchimp costs skyrocketing. This is because Mailchimp's pricing is based on how many emails you send each month, which makes it entirely unsuited for a complex onboarding campaign and left us searching for a platform with more predictable and scalable pricing.
```